### PR TITLE
Feature/research lib revisions

### PR DESCRIPTION
**Pull request by touchRED**
- Adds new route & layout for research library topics
- Updates research library index page to render cards (w/ background images) for each topic
- Updates CMS structure to allow for new layout

### CODE REVIEW

File: src/pages/research-library.js [hunk 1]
```diff
@@ -402,43 +351,101 @@ const SessionsPage = ({ location }) => {
...
-              </div>
-            )
-          })}
-        </Col>
-      </Row> */}
     </Layout>
   )
 }
 
+// const LegacyTopicsBody = () => {
```
Comment (Lane) [inline src/pages/research-library.js:358]: any reason to keep this commented out component?

**Review by Lane — CHANGES_REQUESTED** [review]
trying to run this locally (`npm install` then `npm run develop`), but i am running into some graphql errors.  any ideas?

```
 ERROR #85923  GRAPHQL

There was an error in your GraphQL query:

Cannot query field "label" on type "ResearchItem".

If you don't expect "label" to exist on the type "ResearchItem" it is most
likely a typo.
However, if you expect "label" to exist there are a couple of solutions to
common problems:
```

[log.txt](https://github.com/Hyperobjekt/untd-site/files/6649809/log.txt)

**Comment by touchRED** [comment]
Hey @Lane it looks like a merge conflict somehow got committed (located in `research-library.mdx`). I’ll fix it and push it back up, removing that legacy component as well.

**Comment by touchRED** [comment]
@Lane good to go!

**Comment by Lane** [comment]
@touchRED running now, thanks!

this looks good.  one thing i'm seeing is that when i select a page from the research library, i am scrolled part way down the page. 

image

can we make it so the page is at the top after navigation?

**Comment by Lane** [comment]
@touchRED going to merge this in with the above issue so that UNTD team can start content updates.  will add a separate issue for the above

**Pull request merged**